falco-plugin/seccomp_agent_rules.yaml

```
@@ -0,0 +1,7 @@
+- rule: Seccomp Agent
```

```suggestion
- required_engine_version: 11

- rule: Seccomp Agent
```

Not strictly required, but recommended.

---

Updated, thanks!